internal/envoy/embed.go

```diff
@@ -60,5 +62,10 @@ func extractEmbeddedEnvoy() (outPath string, err error) {
 		_ = os.Chtimes(outPath, zfi.ModTime(), zfi.ModTime())
 	}
 
+	log.Info().
```

Comment:
maybe we should put the debug log statement at the end of the `NewServer` func and print everything we know about the envoy binary (whether or not checksum was set, what the actual checksum was, the path, etc). Thoughts?